### ISSUE TITLE
Close dialog boxes by clicking outside them

### ISSUE THREAD

**Issue by Malikil** [Reyder95/sdvx-tracker-frontend#81]
It would be nice to be able to cancel a dialog box by clicking outside of it, rather than having to click the x in the top right corner.

**Comment by Reyder95** [comment]
I'm aware of this, it's just a pain to do it via react. Have to figure out how to make it work properly. Secondly as for them being centered it's not a huge priority rn to fix that specifically. Since it works correctly nonetheless. Maybe when I do the modal redesign I'll fix that.

**Comment by Malikil** [comment]
Yeah both of these things are just QoL "Nice to have, but not ultimately needed"

**Comment by Reyder95** [comment]
If someone decides to tackle this is might be best to put both modals in components of their own (one already is). Using react hooks it should be doable (or smth like useEffect).
